examples/language-modeling/run_mlm_flax.py

```diff
@@ -548,6 +588,10 @@ def tokenize_function(examples):
 
     # Data collator
     # This one will take care of randomly masking the tokens.
+    if training_args.tokenizers_parallelism:
```

Comment:
Don't like this if-else too much -> the user could just export those in the command line no?